staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6186,6 +6186,15 @@ type NodeSystemInfo struct {
 	OperatingSystem string `json:"operatingSystem" protobuf:"bytes,9,opt,name=operatingSystem"`
 	// The Architecture reported by the node
 	Architecture string `json:"architecture" protobuf:"bytes,10,opt,name=architecture"`
+	// Swap Info reported by the node.
+	Swap *Swap `json:"swap,omitempty" protobuf:"bytes,11,opt,name=swap"`
+}
+
+// Swap represents swap memory information.
+type Swap struct {
```

Comment:
I was previously concerned witht hhe field name.  I think the TYPE can be more precisely named, not that "info" added much but `NodeSwapStatus` or something?

Reply:
Done :+1: